### DISCUSSION TITLE
@heroicons/svelte

### DISCUSSION THREAD

**Discussion by shinokada** [tailwindlabs/heroicons#592] [Ideas]
Hi,

Thank you for awesome icons.
Is it possible  to add Svelte as well?

**Comment by enheit** [comment]
I also would like this library to be wrapped in the svelte package

**Reply by shinokada** [reply]
You can try [Svelte-heros](https://www.npmjs.com/package/svelte-heros)

**Comment by carloslfu** [comment]
I like https://github.com/steeze-ui/icons by @JustinVoitel. It supports more frameworks, and it uses Heroicons 2.0.

**Reply by shinokada** [reply]
You can check https://www.npmjs.com/package/svelte-heros-v2 for Heroicons v2.

**Comment by NatoBoram** [comment]
I also made one: [`@natoboram/heroicons.svelte`](https://github.com/NatoBoram/heroicons.svelte)

It's entirely auto-generated from `heroicons@^2.0.13`. It has `w-5 h-5` and `w-6 h-6` on by default, but you can send `class` to override that.
